tests/snmp/conftest.py

```diff
@@ -95,7 +95,8 @@ def check_redis_output(duthost, key):
 @pytest.fixture(scope="module", autouse=True)
 def enable_queue_counterpoll_type(duthosts):
```

Comment:
This fixture might not good to be set to autouse. @lizhijianrd please create an issue, we need clean up this later.

Reply:
ADO: 30605270